https://issues.redhat.com/browse/ACM-6223 Update create_infra_env.adoc

dockerymick opened the pull request:
https://issues.redhat.com/browse/ACM-6223

dockerymick committed:
Update create_infra_env.adoc

https://issues.redhat.com/browse/ACM-6223

dockerymick committed:
Update create_infra_env.adoc

Adding YAML callouts

cadawson committed:
Minor change example to minimum

cadawson committed:
edit

cadawson reviewed (approved):
Looks good. Thanks for taking that!

dockerymick committed:
Update create_infra_env.adoc

Adding more info to my hidden comment. Modular effort being deprioritized for now. Small team

dockerymick commented:
@cadawson thanks for your approval Chris! Modular effort being deprioritized for now since we have a smaller team. We can revisit this. merging the PR
